packages/react-components/react-combobox/library/src/components/Dropdown/Dropdown.cy.tsx

```diff
@@ -57,3 +57,35 @@ describe('Dropdown controlling open/close state', () => {
     cy.get(listboxSelector).should('not.exist');
   });
 });
+
+describe('Dropdown - clearable', () => {
+  const DropdownComponent = (props: Partial<DropdownProps>) => {
+    const options = ['Cat', 'Dog', 'Ferret', 'Fish', 'Hamster', 'Snake'];
+
+    return (
+      <Dropdown placeholder="Select an animal" {...props}>
+        {options.map(option => (
+          <Option key={option}>{option}</Option>
+        ))}
+      </Dropdown>
+    );
+  };
+
+  it('clear button should be hidden and not focusable by default', () => {
+    mount(<DropdownComponent />);
+
+    const clearButton = cy.get(clearButtonSelector);
+
+    clearButton.should('not.be.visible');
+    clearButton.should('have.attr', 'tabIndex', '-1');
+  });
+
+  it('clear button should be visible and focusable when the "clearable" props is set', () => {
+    mount(<DropdownComponent clearable selectedOptions={['Cat']} />);
+
+    const clearButton = cy.get(clearButtonSelector);
+
+    clearButton.should('be.visible');
+    clearButton.should('have.attr', 'tabIndex', '0');
```

Comment:
As the original issue is about tabbing can we simulate <kbd>Tab</kbd> navigation instead? See `packages/react-examples/src/react/FocusTrapZone/e2e/FocusTrapZone.e2e.tsx`

Reply:
done